src/components/buttons/aprove-dev-request.js

```diff
@@ -23,5 +23,18 @@ module.exports = {
     await interaction.message.thread.setArchived(true);
 
     await interaction[interaction.replied || interaction.deferred ? 'followUp' : 'reply']({ content: "Desenvolvedor Aprovado", ephemeral: true });
+  
+    const webhook = parseWebhookURL(process.env.OFFTOPIC_WEBHOOK);
+    await interaction.client.fetchWebhook(webhook.id, webhook.token)
+        .then((webhook) => {
```

Comment:
`async` pra sugestao abaixo funcionar
```suggestion
        .then(async (webhook) => {
```